paddle/fluid/operators/detail/grpc_client.h

```diff
@@ -159,34 +165,32 @@ class FetchBarrierProcessor : public BaseProcessor {
   std::unique_ptr<sendrecv::SendRecvService::Stub> stub_;
 };
 
-class RPCClient {
+class GRPCClient : public RPCClient {
  public:
-  bool AsyncSendVariable(const std::string& ep,
-                         const platform::DeviceContext& ctx,
-                         const framework::Scope& scope,
-                         const std::string& var_name,
-                         int64_t time_out = 600 * 1000);
-
-  bool AsyncGetVariable(const std::string& ep,
-                        const platform::DeviceContext& ctx,
-                        const framework::Scope& scope,
-                        const std::string& var_name,
-                        int64_t time_out = 600 * 1000);
-
-  bool AsyncPrefetchVariable(const std::string& ep,
-                             const platform::DeviceContext& ctx,
-                             const framework::Scope& scope,
-                             const std::string& in_var_name,
-                             const std::string& out_var_name,
-                             int64_t time_out = 600 * 1000);
-
-  void AsyncSendBatchBarrier(const std::string& ep,
-                             int64_t time_out = 600 * 1000);
-
-  void AsyncSendFetchBarrier(const std::string& ep,
-                             int64_t time_out = 600 * 1000);
-
-  bool Wait();
+  virtual bool AsyncSendVariable(const std::string& ep,
+                                 const framework::Scope& scope,
+                                 const std::string& var_name,
+                                 int64_t time_out = RPCClient::rpc_time_out);
+
+  virtual bool AsyncGetVariable(const std::string& ep,
```

Comment:
I think we should replace virtual with overwrite in subclass
https://stackoverflow.com/questions/39932391/virtual-override-or-both-c

Reply:
Thanks. I will fix it.